llm/memory.go

```diff
@@ -283,7 +283,7 @@ func EstimateGPULayers(gpus []gpu.GpuInfo, ggml *GGML, projectors []string, opts
 		availableList:       availableList,
 		kv:                  kv,
 		allocationsList:     allocationsList,
-		memoryWeights:       memoryWeights,
+		memoryWeights:       memoryWeights - kv,
```

Comment:
Thanks for fixing this

Even if just for logging, but it's not super obvious that `memoryWeights` includes the kv to begin with (I see it's in the layer calculations above). I am wondering if we could re-organize that to avoid memoryWeights ever having the kv amount in it?

Reply:
split the KV out from the weights